staging/src/k8s.io/api/flowcontrol/v1beta1/register.go

```diff
@@ -54,5 +56,15 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&PriorityLevelConfigurationList{},
 	)
 	metav1.AddToGroupVersion(scheme, SchemeGroupVersion)
+
+	// Registers the lifecycle of the group version, which is checked to make sure a gvr is not available before its type is introduced or after it is removed.
```

Comment:
It is unexpected that this function would change to modify global state, it is intended to only modify the passed-in scheme variable. In additional to changing shared global state not being desired, doing so is likely to cause panic / race conditions (similar to https://github.com/kubernetes/kubernetes/issues/123632) or fatally error if two different schemes tried to register this API group into themselves.

(this comment applies to all `addKnownTypes(scheme *runtime.Scheme)` changes)